### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by jsquyres**
* Add a warning to the top of FAQ pages (with the exception of the OMPIO page so that this PR doesn't generate conflicts with #11241 -- FYI @edgargabriel) 
* Add a section about installing binary packages to the quickstart install page
* Remove the MacOS FAQ page; fold relevant content into other parts of the docs.
* Remove the redundant OPA networking page; its content was already included in the OFI networking page.

### CODE REVIEW

**Comment by jsquyres** [comment]
@edgargabriel Can you review?

File: docs/tuning-apps/mpi-io/ompio.rst [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+   operation will be executed as one chunk. Splitting the operation up
+   into multiple, smaller chunks can lead to performance improvements
+   in certain scenarios.
+
+#. ``io_ompio_bytes_per_agg``: Size of temporary buffer for collective
+   I/O operations on aggregator processes. Default value is 32MB.
+   Tuning this parameter has a very high impact on the performance of
+   collective operations.
+
+   .. note:: Be sure to also see recommendations for tuning collective
```
Comment (jsquyres) [inline docs/tuning-apps/mpi-io/ompio.rst:134]: @edgargabriel What section are you referring to here?  We should put in a `:ref:` cross reference.

File: docs/tuning-apps/mpi-io/ompio.rst [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+             operations.
+
+#. ``io_ompio_num_aggregators``: Number of aggregators used in
+   collective I/O operations.  Setting this parameter to a value
+   larger zero disables the internal automatic aggregator selection
+   logic of OMPIO.  Tuning this parameter has a very high impact on
+   the performance of collective operations.
+
+   .. note:: Be sure to also see recommendations for tuning collective
```
Comment (jsquyres) [inline docs/tuning-apps/mpi-io/ompio.rst:143]: @edgargabriel Similar to above: what section are you referring to here?  We should put in a `:ref:` cross reference.

File: docs/tuning-apps/mpi-io/ompio.rst (hunk 1 above)
Comment (edgargabriel) [reply docs/tuning-apps/mpi-io/ompio.rst:134]: will have to remove that, we used to have separate section on tuning collectives, but I decided to remove it, was more confusing than helpful

File: docs/tuning-apps/mpi-io/ompio.rst (hunk 2 above)
Comment (edgargabriel) [reply docs/tuning-apps/mpi-io/ompio.rst:143]: same here, need to remove the note, I overlooked that there was still a cross-reference to a removed section

Comment (edgargabriel) [reply docs/tuning-apps/mpi-io/ompio.rst:143]: Shall I file a pr against your branch to fix these two references, or do you want to do that?

**Review by edgargabriel — APPROVED** [review]
other than the two incorrect references in the ompio section to non-existent sub-sections, things look good to me.

File: docs/tuning-apps/mpi-io/ompio.rst [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+  will write 1/n th of the overall file.
+
+* ``individual``: this components executes all collective I/O
+  operations in terms of individual I/O operations.
+
+The ``sharedfp`` framework provides a different implementation of the
+shared file pointer operations depending on file system features.
+
+* ``lockfile``: this component will be used on file system which
+  support for file locking.
```
Comment (edgargabriel) [inline docs/tuning-apps/mpi-io/ompio.rst:104]: re-reading I noticed a small issue here: it should probably be 'this component will be used on file systems which support file locking' (plural for file syste(s) and should remove 'for')

File: docs/tuning-apps/mpi-io/ompio.rst [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+   ``stripe_width`` MPI Info value.
+
+Using GPU device buffers in MPI File I/O operations
+----------------------------------------------------
+
+OMPIO supports reading and writing directly to/from GPU buffers using
+the MPI I/O interfaces. Using this feature simplifies managing buffers
+that are exclusive used on GPU devices, and hence the necessity to
+implement a staging through host memory for file I/O operations.
+
```
Comment (edgargabriel) [inline docs/tuning-apps/mpi-io/ompio.rst:195]: proof-reading: '... that are exclusively used on GPU devices, and hence there is no need to implement staging through host memory for file I/O operations'

**Comment by edgargabriel** [comment]
@jsquyres since I had 2-3 other smaller changes as well, I decided to file a pr against your branch to simplify things

**Comment by jsquyres** [comment]
> @jsquyres since I had 2-3 other smaller changes as well, I decided to file a pr against your branch to simplify things

Merged -- thanks!

**Pull request merged**